src/footprintmonitoring/azext_footprintmonitoring/generated/_help.py

```diff
@@ -0,0 +1,238 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+
+helps['footprintmonitoring profile'] = """
+    type: group
+    short-summary: footprintmonitoring profile
+"""
+
+helps['footprintmonitoring profile list'] = """
+    type: command
+    short-summary: Retrieves the information about all Footprint profiles under a subscription.
+    examples:
+      - name: List Footprint profiles in a Resource Group
+        text: |-
+               az footprintmonitoring profile list --resource-group "rgName"
+"""
+
+helps['footprintmonitoring profile show'] = """
+    type: command
+    short-summary: Retrieves the information about a single Footprint profile.
+    examples:
+      - name: profilesGet
+        text: |-
+               az footprintmonitoring profile show --name "fpProfile1" --resource-group "rgName"
+"""
+
+helps['footprintmonitoring profile create'] = """
+    type: command
+    short-summary: Creates or updates a Footprint profile with the specified properties.
+    examples:
+      - name: profilesCreateOrUpdate
+        text: |-
+               az footprintmonitoring profile create --location "westus2" --measurement-count 3 \
+--start-delay-milliseconds 5000 --tags key1="value1" key2="value2" --name "fpProfile1" --resource-group "rgName"
+"""
+
+helps['footprintmonitoring profile update'] = """
+    type: command
+    short-summary: Updates an existing Footprint profile resource.
+    examples:
+      - name: profilesUpdate
+        text: |-
+               az footprintmonitoring profile update --tags key1="value1" key2="value2" --name "fpProfile1" \
+--resource-group "rgName"
+"""
+
+helps['footprintmonitoring profile delete'] = """
+    type: command
+    short-summary: Deletes an existing Footprint profile.
+    examples:
+      - name: profilesDelete
+        text: |-
+               az footprintmonitoring profile delete --name "fpProfile1" --resource-group "rgName"
+"""
+
+helps['footprintmonitoring measurement-endpoint'] = """
+    type: group
+    short-summary: footprintmonitoring measurement-endpoint
```

Comment:
Same here. @qiaozha Do you know how to control this short-summary for command group in manual folder?

Reply:
what do you mean by control ? overwrite it?

Reply:
yes. overwriting it.

Reply:
If you need to overwrite the help info you can add a _help.py in manual folder then put the part you want to overwrite.

Reply:
thanks. I overwrote these.
see equivalent file under azext_footprint/*